src/Tools/Source/RunTests/Program.cs

```diff
@@ -358,6 +369,15 @@ static bool shouldExclude(string name, Options options)
 
                 return false;
             }
+
+            static bool IsMatch(TestTargetFramework testTargetFramework, string dirName) =>
+                testTargetFramework switch
+                {
+                    TestTargetFramework.Both => true,
+                    TestTargetFramework.Core => Regex.IsMatch(dirName, @"^net[\d]+\."),
+                    TestTargetFramework.Framework => dirName is "net472",
```

Comment:
```suggestion
                    TestTargetFramework.Framework => dirName is "net4\d\d?",
```

I'm not sure if we'd ever have a reason to upgrade to net48 but if we do then this won't break that.

Reply:
I see now you already said this to @dibarbet too, but it was collapsed. :smile: